### PR TITLE
Revert "Feature/is admin middleware"

### PR DESCRIPTION
**Pull request by felmez**
Reverts RCDD-202110-TUR-BEW/backend-capstone-turkey-bursapediary#17

### CODE REVIEW

File: src/middlewares/isAdmin.js [hunk 1]
```diff
@@ -1,21 +0,0 @@
-const jwt = require('jsonwebtoken');
```
Comment (halitbatur) [inline src/middlewares/isAdmin.js:1]: why are u deleting the isAdmin file?

**Review by halitbatur — CHANGES_REQUESTED** [review]

File: src/middlewares/isAdmin.js (hunk 1 above)
Comment (felmez) [reply src/middlewares/isAdmin.js:1]: i am not deleting its replaced with the isLogged but now it should be fine both

**Review by halitbatur — APPROVED** [review]

File: src/middlewares/isAdmin.js (hunk 1 above)
Comment (Amr-Nash) [reply src/middlewares/isAdmin.js:1]: I think @felmez you should write both the isAdmin and isLogedIn files in their latest state in this branch and then push

Comment (felmez) [reply src/middlewares/isAdmin.js:1]: already did this, thanks

**Pull request merged**